src/publisher/models.py

```diff
@@ -266,44 +266,15 @@ class Meta:
         # lowest positions first (ASC), earliest creation date first (ASC)
         ordering = ('position', 'datetime_record_created')
 
+DATE_EJP_QC, DATE_EJP_DECISION = 'date-ejp-qc', 'date-ejp-decision'
```

Comment:
can we give names to these fields/events that abstract EJP away from the model? Lax could be usable with any peer review system, not just with EJP